rclcpp/test/rclcpp/node_interfaces/test_node_services.cpp

```diff
@@ -129,3 +132,13 @@ TEST_F(TestNodeService, add_client_rcl_trigger_guard_condition_error)
     node_services->add_client(client, callback_group),
     std::runtime_error("Failed to notify wait set on client creation: error not set"));
 }
+
+TEST_F(TestNodeService, resolve_service_name)
+{
+  EXPECT_EQ("/ns/bar", node_services->resolve_service_name("foo", false));
+  EXPECT_EQ("/ns/foo", node_services->resolve_service_name("foo", true));
+  EXPECT_EQ("/foo", node_services->resolve_service_name("/foo", true));
```

Comment:
For completeness, consider trying to other option too
```
  EXPECT_EQ("/foo", node_services->resolve_service_name("/foo", false));
```

Reply:
But in this case the result will be "/bar", right?
I can add it, but the test in line 138 seems to be covering that case.

Reply:
ah, yeah you're right. Adding an extra test doesn't seem necessary.